src/stcal/ramp_fitting/likely_fit.py

```diff
@@ -0,0 +1,1353 @@
+#! /usr/bin/env python
+
+import logging
+import multiprocessing
+import time
+import scipy
+import sys
+import warnings
+
+from multiprocessing import cpu_count
+from pprint import pprint
+
+import numpy as np
+
+from . import ramp_fit_class, utils
+from .likely_algo_classes import IntegInfo, Ramp_Result, Covar
+
+
+DELIM = "=" * 80
+SQRT2 = 1.41421356
```

Comment:
I purposefully remove all unnecessary computations.  The entire purpose of the use of `SQRT2` is that it is NOT a magic number.  It is properly labelled as the square root of 2.

Reply:
It doesn't add much to processing time to compute the square root of 2 once.  I'm fine with keeping the constant, but I think it would improve clarity to set the value to np.sqrt(2).

Reply:
I support np.sqrt(2) here.